docassemble/MAVirtualCourt/data/questions/Housing_Temporary_Restraining_Order.yml

```diff
@@ -251,14 +251,35 @@ fields:
     datatype: yesno
   - 'Explain what utility was shut off, and when': incident_utility_description
     show if: incident_utility
-  - 'Other breach of quiet enjoyment': incident_other
+  - 'Other (breach of my quiet enjoyment)': incident_other
     datatype: yesno
   - 'Describe the other thing your landlord did wrong': incident_other_description
     datatype: area
     show if: incident_other
+    help: |
+      You can include anything here that is a breach of your right to
```

Comment:
Suggestion:
> Anything that makes you unable to be in your home safely and peacefully counts as a breach of your right to "quiet enjoyment". For example: an unsafe living condition or a smell or noise that makes your home unlivable.

There are other ways I'd love to play with the language too, but that would at least avoid burying the most important part at the end.